Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * File:   FixedWing.c
```

Comment:
Add doxygen style file headers if you're creating a new file. In this case, @file, @author, etc. Look at examples in my PR